README.md

```diff
@@ -195,6 +195,7 @@ A sample configuration file with all options is available [here](https://github.
     * `"tabs"` enforces consistent tabs.
     * `"spaces"` enforces consistent spaces.
 * `interface-name` enforces the rule that interface names must begin with a capital 'I'
+    * `"no-prefix"` enforces the rule that interface names must not have a 'I' prefix
```

Comment:
Can we make the options `always-prefix` and `never-prefix`? Sort of following the same pattern as [this PR](https://github.com/palantir/tslint/pull/974/files). If neither option is provided, we'll default to `always-prefix` to maintain backwards compatibility.